JRFMemoryNoodler/JRFMemoryNoodler.m

```diff
@@ -10,30 +10,43 @@
 
 #import "JRFMemoryNoodler.h"
 
+#include <sys/stat.h>
+
 NSString *JRFPreviousBundleVersionKey = @"JRFPreviousBundleVersionKey";
 NSString *JRFAppWasTerminatedKey = @"JRFAppWasTerminatedKey";
 NSString *JRFAppWasInBackgroundKey = @"JRFAppWasInBackgroundKey";
 NSString *JRFAppDidCrashKey = @"JRFAppDidCrashKey";
 NSString *JRFPreviousOSVersionKey = @"JRFPreviousOSVersionKey";
-NSString *JRFAppDidAbortKey = @"JRFAppDidAbortKey";
+static char *intentionalQuitPathname;
 
 @implementation JRFMemoryNoodler
 
 + (void)beginMonitoringMemoryEventsWithHandler:(JRFOutOfMemoryEventHandler)handler
                                  crashDetector:(JRFCrashDetector)crashDetector {
-    
+
     [[self sharedInstance] beginApplicationMonitoring];
     signal(SIGABRT, JRFIntentionalQuitHandler);
     signal(SIGQUIT, JRFIntentionalQuitHandler);
     NSUserDefaults *defaults = [NSUserDefaults standardUserDefaults];
-    
+    NSArray *paths = NSSearchPathForDirectoriesInDomains(NSApplicationSupportDirectory, NSUserDomainMask, YES);
+
+    // Set up the static path for intentional aborts
+    NSString *appSupportDirectory = [paths objectAtIndex:0];
+    NSString *fileName = [NSString stringWithFormat:@"%@/intentionalquit.txt", appSupportDirectory];
```

Comment:
I'd prefer to not make this a .txt if we're not actually going to write any text into it. Also, can you use `[appSupportDirectory stringByAppendingPathComponent:...]` instead?